.github/workflows/format.yaml

```diff
@@ -14,7 +14,7 @@ on:
 jobs:
   pre-commit:
     name: Format
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
Revert this? I thought you had updated in another PR, weird.

Reply:
Thanks. Robert wasn't a fan of that change so I'll leave it be. Will revert it.

Don't merge yet, I'm going to try one more thing.